### ISSUE TITLE
What does MPW stand for?

### ISSUE THREAD

**Issue by DeflateAwning** [efabless/mpw_precheck#268]
Please add it somewhere within this repo. Acronyms without definitions are the worst.

**Comment by d-mitch-bailey** [comment]
@DeflateAwning I agree, excessive use of TLA's (Three Letter Acronyms) can be annoying. ;)

However, MPW is pretty standard in foundry parlance. It stands for Multi Project Wafer - where multiple projects are fabricated on the same wafer. MPW's are also often referred to as shuttles.

**Comment by DeflateAwning** [comment]
I understand; I know that now, after Googling for it. I still think that any acronym in a repo's name should be defined somewhere in the readme, hence this request.
